src/common/components/layouts/pages/inspect/validationErrorMessages_technical.json

```diff
@@ -298,8 +298,20 @@
 				"DESCRIPTION": "The PDF/UA version and conformance level of a file shall be specified using the PDF/UA Identification extension schema"
 			},
 			"2": {
-				"SUMMARY": "The \"part\" property of the PDF/UA Identification Schema is %1 instead of 1 for PDF/UA conforming file",
+				"SUMMARY": "The \"part\" property of the PDF/UA Identification Schema is not 1 for PDF/UA-1 conforming file",
 				"DESCRIPTION": "The value of pdfuaid:part shall be the part number of the International Standard to which the file conforms"
+			},
+			"3": {
+				"SUMMARY": "Property part of the PDF/UA Identification Schema has an invalid namespace prefix",
```

Comment:
Put part, amd, corr properties in quotes in this and the next two error messages